### DISCUSSION TITLE
/dev/fd/63: line 30: SPINNER_PID: unbound variable

### DISCUSSION THREAD

**Discussion by nicoska84** [tteck/Proxmox#2533] [Help Needed]
I'm getting this error message every time I try to run a script. It worked flawless yesterday :/

`Loading...
bash: line 65: start: command not found
/dev/fd/63: line 30: SPINNER_PID: unbound variable
root@nicoska2:~# `

**Comment by mattbrothwell** [comment]
Getting the exact same error message when running the Tdarr script, script was updated 5hrs ago, think a typo might have happened.

**Comment by tteck** [comment]
An important update broke things for a minute, all good now.
